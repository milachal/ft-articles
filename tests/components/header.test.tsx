import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import Header from "../../components/header";

jest.mock("../../components/header/header.module.scss", () => <div />);
jest.mock("../../components/searchBar/searchBar.module.scss", () => <div />);
jest.mock("../../components/searchBar/search-suggestion.module.scss", () => <div />);

describe("Header", () => {
  it("should render correctly", () => {
    const wrapper: ShallowWrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
