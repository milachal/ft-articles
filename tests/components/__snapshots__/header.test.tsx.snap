// Jest Snapshot v1, https://goo.gl/fbAQLP

exports["Header should render correctly 1"] = `
<div>
  <div>
    <Search />
  </div>
  <Link
    href="/"
  >
    <a>
      <div>
        <img
          alt="financial-times-logo"
          src="https://www.ft.com/__origami/service/image/v2/images/raw/ftlogo:brand-ft-masthead?format=svg&source=page-kit-layout&tint=%2333302E%2C%2333302E"
        />
      </div>
    </a>
  </Link>
</div>
`;
